webapp/src/main/webapp/src/app/pages/tests/create/test-create.page.ts

```diff
@@ -24,9 +24,10 @@ import { Notification, NotificationEvent, NotificationService, NotificationType
 import { ServicesService } from '../../../services/services.service';
 import { TestsService } from '../../../services/tests.service';
 import { SecretsService } from '../../../services/secrets.service';
-import { Operation, Service } from '../../../models/service.model';
+import {Contract, Operation, Service} from '../../../models/service.model';
 import { TestRunnerType, OAuth2ClientContext } from '../../../models/test.model';
 import { Secret } from '../../../models/secret.model';
+import {ContractsService} from "../../../services/contracts.service";
```

Comment:
Could you please follow the same format? => `{ ContractsService }`
and put this import within the "services" block? => insert it at line 24

Reply:
Made this change.